catkin_home/src/arm_vision/src/example1.cpp

```diff
@@ -0,0 +1,94 @@
+#include <ProcessingCloud.h>
+
+class segmentation {
```

Comment:
segmentation -> Segmentation

Reply:
if possible put this class in a separate file Segmentation.h and Segmentation.cpp